lighthouse-core/config/config.js

```diff
@@ -459,22 +459,23 @@ class Config {
 
   /**
    * @param {LH.Config.Json} configJSON
+   * @param {LH.Flags=} flags
    * @param {string=} configDir
    * @return {LH.Config.Json}
    */
-  static mergePlugins(configJSON, configDir) {
-    const pluginNames = configJSON.plugins;
+  static mergePlugins(configJSON, flags, configDir) {
```

Comment:
these are like the only meaningful changes, right?

so clean ✨ 😃

Reply:
> these are like the only meaningful changes, right?

yep, that's it! soooo much better

Reply:
haha, I also just realized this was the original plan all along, so at least we got back to it :)

> so it seems ok to add this to the handful of special-cased flag/config mergings in config.js when we add support for the CLI flag.

https://github.com/GoogleChrome/lighthouse/pull/6959#issuecomment-452540140